cms/src/main/java/uk/nhs/hee/web/taxonomy/TaxonomySingleFieldPlugin.html

```diff
@@ -0,0 +1,48 @@
+<!--
+    * Copyright 2023 Bloomreach
+    *
+    * Licensed under the Apache License, Version 2.0 (the  "License");
+    * you may not use this file except in compliance with the License.
+    * You may obtain a copy of the License at
+    *
+    * http://www.apache.org/licenses/LICENSE-2.0
+    *
+    * Unless required by applicable law or agreed to in writing, software
+    * distributed under the License is distributed on an "AS IS" BASIS,
+    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    * See the License for the specific language governing permissions and
+    * limitations under the License.
+-->
+<html xmlns:wicket="http://wicket.apache.org/">
+    <wicket:panel>
+    <div class="hippo-editor-field hippo-property-field-name-string hippo-property-field-type-string hippo-property-field-multiple">
+        <h3 class="hippo-editor-field-title">
+            <span wicket:id="title">[taxonomy]</span>
+			      <span class="required" wicket:id="required">*</span>
```

Comment:
Can we put `(<span wicket:id="taxonomy-name">taxonomy name</span>)` back in order to be inline with other taxonomy fields unless it is removed intentionally ?

Reply:
I've removed that span because it makes the the field look ugly with the parenthesis !

Reply:
Sure, I understand. Basically, I just wanted to put it back for consistency with other taxonomy fields which currently display the taxonomy. How about we put it back for now and check with Kat/Content team for their opinion whether to remove or keep it ? thanks

Reply:
I'll remove the file entirely as it's just an override and the ancestor class's HTML file will now get picked up in preference